ui/components/app/confirm/info/info.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { ConfirmInfoRow } from '../shared/info/row';
+import { ConfirmInfo, ConfirmInfoRowConfig, ConfirmInfoRowType } from './info';
+
+const mockRowConfigs: ConfirmInfoRowConfig[] = [
+  {
+    label: 'Address',
+    variant: ConfirmInfoRowType.Address,
+    rowProps: {
+      address: '0xCcCCccccCCCCcCCCCCCcCcCccCcCCCcCcccccccC',
+    },
+  },
+  {
+    variant: ConfirmInfoRowType.Divider,
+  },
+  // todo: replace row variant for Message. Possible value or value-expandable. neither have been created yet
+  {
+    label: 'Message',
+    variant: ConfirmInfoRowType.ValueDouble,
+    rowProps: {
+      left: '',
+      right:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vel suscipit tortor. Curabitur vulputate felis nibh, vel pellentesque erat fermentum eget. Duis id turpis cursus, blandit magna sit amet, tempor sem. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Maecenas ex nulla, suscipit id eros in, elementum lacinia leo. Etiam dignissim neque vitae nibh pretium, sed egestas est mollis. Nam venenatis tellus sed tempor bibendum.',
+    },
+  },
+];
+
+const ConfirmInfoStory = {
+  title: 'Components/App/Confirm/Info',
+  component: ConfirmInfoRow,
+
+  args: { rowConfigs: [...mockRowConfigs] },
+  argTypes: {
+    rowConfigs: {
+      control: {
+        type: 'object',
+      },
+    },
+  },
+};
+
+export const DefaultStory = (args) => <ConfirmInfo {...args} />;
+
+DefaultStory.storyName = 'Default';
+
+export default ConfirmInfoStory;
```

Comment:
been quite puzzled by this issue... continuing to debug tomorrow

![Screenshot 2023-12-05 at 7 09 09 PM](https://github.com/MetaMask/metamask-extension/assets/20778143/ba0e63af-eeef-44c4-b897-a5429eb5b7d6)